### ISSUE TITLE
API is unstable

### ISSUE THREAD

**Issue by AndreDK7** [TTV-LOL/extensions#54]
TTV.LOL API is being unstable for the past 2 weeks. Sometimes is online but after a minute is giving error 520 and then coming back. It's being like this since 2 weeks ago.

**Comment by Chassss** [comment]
yeah it really needs to get fixed, at this point literally watching ads is more time efficient than buffering for 5 minutes then having a ad play because the redirect to the api fails, or when it doesnt fail it'll just buffer for 30 seconds then play your stream then when a ad comes on it'll rinse repeat

**Comment by PatientXero** [comment]
Don't expect them to do anything. It's an unpaid project and like anything that's free, they'll just dump it and leave us to use an alternative.
